sw/device/lib/dif/dif_usbdev.c

```diff
@@ -560,12 +561,45 @@ dif_result_t dif_usbdev_send(const dif_usbdev_t *usbdev, uint8_t endpoint,
   return kDifOk;
 }
 
+dif_result_t dif_usbdev_get_tx_sent(const dif_usbdev_t *usbdev,
+                                    uint16_t *sent) {
+  if (usbdev == NULL || sent == NULL) {
+    return kDifBadArg;
+  }
+  *sent = mmio_region_read32(usbdev->base_addr, USBDEV_IN_SENT_REG_OFFSET);
+  return kDifOk;
+}
+
+dif_result_t dif_usbdev_clear_tx_status(const dif_usbdev_t *usbdev,
+                                        dif_usbdev_buffer_pool_t *buffer_pool,
+                                        uint8_t endpoint) {
+  if (usbdev == NULL || buffer_pool == NULL || !is_valid_endpoint(endpoint)) {
+    return kDifBadArg;
+  }
+  // Get the configin register offset and bit index of the endpoint
```

Comment:
All the other comments in this file finishes with a period, could you do the same of the new comments for consistency?